Review section for Chart Control. (#609)
telerik/maui-docs: controls/chart/series/pie/pie-series.md

title: Pie Series
page_title: .NET MAUI Chart Documentation - Pie Series
slug: chart-series-pie-series

> description: Learn more about the Pie series in Telerik UI for .NET MAUI Chart control.

Reviewer: - Use the definite article.

The Pie Series supports the following properties:

- `ValueBinding`&mdash;Defines the binding to a property of the data model that will be used to fill the pie slices.

> - `RadiusFactor`&mdash;Defines the radius factor used to calculate the radius of the visual series. This value is usually within the [0,1] range but you can oversize the series by setting a value greater than 1.

Reviewer: ```suggestion
- `RadiusFactor`&mdash;Defines the radius factor used to calculate the radius of the visual series. This value is usually within the [`0,1`] range but you can oversize the series by setting a value greater than `1`.
```

> - `RadiusFactor`&mdash;Defines the radius factor used to calculate the radius of the visual series. This value is usually within the [0,1] range but its possible to oversize the series by setting a value greater than 1.

Reviewer: ```suggestion
- `RadiusFactor`&mdash;Defines the radius factor used to calculate the radius of the visual series. This value is usually within the [0,1] range but you can oversize the series by setting a value greater than 1.
```
